starlite/cli/commands/openapi.py

```diff
@@ -0,0 +1,53 @@
+from json import dumps
+from pathlib import Path
+
+from click import group, option
+from jsbeautifier import Beautifier
+from yaml import dump as dump_yaml
+
+from starlite import Starlite
+from starlite.cli.utils import StarliteCLIException, StarliteGroup
+from starlite.openapi.typescript_converter.converter import (
+    convert_openapi_to_typescript,
+)
+
+beautifier = Beautifier()
+
+
+@group(cls=StarliteGroup, name="openapi")
+def openapi_group() -> None:
+    """Manage server-side openapi."""
+
+
+@openapi_group.command("schema")
+@option("--output", help="output file path", type=str, default="openapi_schema.json")
```

Comment:
```suggestion
@option("--output", help="output file path", type=click.Path(), default="openapi_schema.json", show_default=True)
```
Ensures that it's a valid path and shows default